react/StyleGuideProvider/StyleGuideProvider.js

```diff
@@ -7,6 +7,10 @@ import classnames from 'classnames';
 import Helmet from 'react-helmet';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
+if (typeof window !== 'undefined') {
+  require('smoothscroll-polyfill').polyfill();
```

Comment:
Good point, actually. At the very least, we could only import this in components that actually use it?